app/models/phylo_tree.rb

```diff
@@ -253,7 +253,8 @@ def prepare_dag(dag_name, attribute_dict)
                            attribute_dict,
                            parse_dag_vars)
     self.dag_json = dag.render
-    upload_dag_json_and_return_job_command(dag_json, dag_s3, dag_name)
```

Comment:
Since this method is split into two (see comment in changes for `pipeline_run_helpers`), I just call the two methods in the same order they initially appeared in the original function.